docgen/src/Getting_started_React_native.md

```diff
@@ -0,0 +1,642 @@
+
+This connector gives you one interesting properties: 
+
+* `highlight`, a function that returns an array of parsed hits indicating if they are highlighted or not. 
+
+When using this component you will have to provide two properties: 
+
+* `attributeName` attribute name from the record that you want to highlight. 
+* `hit` , the hit to highlight
+
+```jsx 
+// We need to import the connectHighlight to our import
+import {
+  connectInfiniteHits,
+  connectSearchBox,
+  connectHighlight,
+} from 'react-instantsearch/connectors';
+
+const Highlight = connectHighlight(
+  ({ highlight, attributeName, hit, highlightProperty }) => {
+    const parsedHit = highlight({
+      attributeName,
+      hit,
+      highlightProperty: '_highlightResult',
+    });
+    const highlightedHit = parsedHit.map((part, idx) => {
+      if (part.isHighlighted)
+        return (
+          <Text key={idx} style={{ backgroundColor: '#ffff99' }}>
+            {part.value}
+          </Text>
+        );
+      return part.value;
+    });
+    return <Text>{highlightedHit}</Text>;
+  }
+);
+```
+
+Let’s use this new `<Highlight/>` component inside our `<Hits/>` template: 
+
+```jsx
+// [...]
+
+const Hits = connectInfiniteHits(({ hits, hasMore, refine }) => {
+  
+ // [...]
+
+  return (
+    <FlatList
+      data={hits}
+      onEndReached={onEndReached}
+      keyExtractor={keyExtractor}
+      renderItem={({ item }) => {
+        return (
+          <View style={{ flexDirection: 'row', alignItems: 'center' }}>
+            <Image
+              style={{ height: 100, width: 100 }}
+              source={{ uri: item.image }}
+            />
+            <View style={{ flex: 1 }}>
+              <Text>
+                <Highlight attributeName="name" hit={item} />
+
+              </Text>
+              <Text>
+                <Highlight attributeName="type" hit={item} />
+
+              </Text>
+            </View>
+          </View>
+        );
+      }}
+    />
+  );
+});
+```
+
+Now the search displays the results and highlights the part of the hit attribute that matches the query. This pattern is very important in search, especially with Algolia, so that the user knows what's going on. This way the search experience becomes a dialogue between the user and the data.
+
+In this part, we've seen the following:
+- how to add a SearchBox to make queries into the records
+- how to highlight the matched part of the results
+- the importance of highlighting in a text-based search
+
+## Filter your results
+
+While the SearchBox is the way to go when it comes to textual search, you
+may also want to provide filters based on the structure of the records.
+
+Algolia provides a set of parameters for filtering by facets, numbers or geo
+location. Instantsearch packages those into a set of widgets and connectors.
+
+Since the dataset used here is an e-commerce one, let's add a RefinementList using the [connectRefinementList](connectors/connectRefinementList.html) connector
```

Comment:
I would slightly change the order to make the link between the fact that it is an e-commerce-related index and the fact that we filter by categories even clearer. Something like: "Since the dataset used here is an e-commerce one, let's filter the products by categories using a RefinementList and its [connectRefinementList](connectors/connectRefinementList.html) connector"